README.rst

```diff
@@ -45,11 +56,39 @@ upon by some of the world's leading scientists and engineers. If you need to
 manipulate numbers on a computer and display or publish the results, give
 SciPy a try!
 
-For the installation instructions, see INSTALL.rst.txt_.
+For the installation instructions, see `our install
+guide <https://scipy.github.io/devdocs/getting_started.html#installation>`__.
+
+
+Call for Contributions
+----------------------
+
+The SciPy project welcomes your expertise and enthusiasm!
+
+Small improvements or fixes are always appreciated; issues labeled as "good
+first issue" may be a good starting point. Have a look at `our contributing
+guide <http://scipy.github.io/devdocs/dev/hacking.html>`__.
+
+Writing code isn’t the only way to contribute to SciPy. You can also:
+
+- review pull requests
+- triage issues
+- develop tutorials, presentations, and other educational materials
+- maintain and improve `our website <https://github.com/scipy/scipy.org>`__
+- develop graphic design for our brand assets and promotional materials
+- translate website content
+- help with outreach and onboard new contributors
+- write grant proposals and help with other fundraising efforts
+
+If you’re unsure where to start or how your skills fit in, reach out! You can
```

Comment:
We don't want meta issues in the issues. Probably we can use the "Discussions" instead or the mailing list.

Reply:
Then I propose to remove `opening a new issue or `. Then if says, either use the mailing list or comment on issues.